makefiles/app_dirs.inc.mk

```diff
@@ -9,6 +9,7 @@ RIOTBASE ?= .
 # 4. sort
 APPLICATION_DIRS := $(sort $(patsubst ./%,%,$(patsubst %/,%,$(dir $(wildcard \
 	$(RIOTBASE)/fuzzing/*/Makefile     \
+	$(RIOTBASE)/boards/*/examples/*/Makefile \
```

Comment:
I would not put example applications in the boards directory. There are already the `examples` and tests` directories for that.

Reply:
But that example is specific to the board.
That was a bit my intention with this PR - introduce board example applications that showcase some more complex applications to demo the features of a board.

E.g. we could also have one for `stm32f429i-disco` to showcase use of the touch screen/display/gyro that is on the board.
Sure we have tests for the individual components, but it's also good to have an example of how it all comes together.

Reply:
Let's not add this here for the moment. There are already `tests/board_microbit` and `tests/board_calliope-mini` with the same purpose.
But I agree that they are a a bit hidden in the tests. Anyway this should be part of a more general discussion and rework of how applications are organized in RIOT.